DOCKER/docker-entrypoint.sh

```diff
@@ -52,14 +51,14 @@ if [ ! -d "$TMHOME/config" ]; then
 fi
 
 # Start tenderdash in the background
-tenderdash "$@" &
+tenderdash "$@" 2>&1 &
```

Comment:
Are you sure you want to send all stderr from Tenderdash? We should probably let Tenderdash control output and adjust the logger to use stdout instead of stderr. There might be some confusion in the future when you do some stderr output from Tenderdash and do not see it when you run a docker container.

Reply:
My AI junior says:

```
For server software running on Linux, it is generally recommended to send logs to `stderr`. This is because `stdout` is typically used for regular output, while `stderr` is used for logging errors and diagnostic information. Sending logs to `stderr` helps to separate log messages from the normal output of the program, making it easier to manage and analyze logs.

Here's a summary of the best practices:

- **stdout**: Use for regular program output, such as the results of a command or data that the program is designed to produce.
- **stderr**: Use for logging errors, warnings, and diagnostic information.

By following this convention, you can ensure that logs are properly handled and can be redirected or processed separately from the main output of your server software.
```

So, Tenderdash behavior to send logs to stderr is **correct**. For a particular use case of Docker, I decided to join these streams for usability.